# HG changeset patch
# User Peter Kihlstedt <peter@kihlstedt.se>
# Date 1401623642 -7200
#      Sun Jun 01 13:54:02 2014 +0200
# Node ID 0571c4ac2c11f37764919c91c2a12a101ec64436
# Parent  4e799881da645ba77fa4dd8082599ef1cfe66230
Remaining missing for Seamonkey, will benfit others also

diff --git a/dom/chrome/dom/dom.properties b/dom/chrome/dom/dom.properties
--- a/dom/chrome/dom/dom.properties
+++ b/dom/chrome/dom/dom.properties
@@ -44,17 +44,16 @@ FormValidationStepMismatchOneValue=Välj
 FormValidationBadInputNumber=Skriv in ett nummer.
 GetAttributeNodeWarning=Bruket av getAttributeNode() är förlegat. Använd getAttribute() i stället.
 SetAttributeNodeWarning=Bruket av setAttributeNode() är förlegat. Använd setAttribute() i stället.
 GetAttributeNodeNSWarning=Bruket av getAttributeNodeNS() är förlegat. Använd getAttributeNS() i stället.
 SetAttributeNodeNSWarning=Bruket av setAttributeNodeNS() är förlegat. Använd setAttributeNS() i stället.
 RemoveAttributeNodeWarning=Bruket av removeAttributeNode() är förlegat. Använd removeAttribute() i stället.
 CreateAttributeWarning=Bruket av document.createAttribute() är förlegat. Använd element.setAttribute() i stället.
 CreateAttributeNSWarning=Bruket av document.createAttributeNS() är förlegat. Använd element.setAttributeNS() i stället.
-SpecifiedWarning=Bruket av egenskapen specified i attribut är förlegat. Det returnerar alltid true.
 OwnerElementWarning=Bruket av egenskapen ownerElement i attribut är förlegat.
 NodeValueWarning=Bruket av egenskapen nodeValue i attribut är förlegat. Använd value i stället.
 TextContentWarning=Bruket av egenskapen textContent i attribut är förlegat. Använd value i stället.
 EnablePrivilegeWarning=Bruket av enablePrivilege är förlegat. Använd i stället kod som körs med system-principalen (t.ex. ett tillägg).
 nsIJSONDecodeDeprecatedWarning=nsIJSON.decode är förlegat. Använd JSON.parse i stället.
 nsIJSONEncodeDeprecatedWarning=nsIJSON.encode är förlegat. Använd JSON.stringify i stället.
 nsIDOMWindowInternalWarning=Bruket av nsIDOMWindowInternal är förlegat. Använd nsIDOMWindow i stället.
 InputEncodingWarning=Bruket av inputEncoding är förlegat.
@@ -116,18 +115,16 @@ ComponentsWarning=Objektet Components ä
 PluginHangUITitle=Varning: insticksmodulen svarar inte
 PluginHangUIMessage=%S kan vara upptagen eller ha slutat svara. Du kan stoppa insticksmodulen nu, eller så kan du fortsätta för att se om insticksmodulen blir klar.
 PluginHangUIWaitButton=Fortsätt
 PluginHangUIStopButton=Stoppa insticksmodul
 # LOCALIZATION NOTE: Do not translate "mozHidden", "mozVisibilityState", "hidden", or "visibilityState"
 PrefixedVisibilityApiWarning=”mozHidden” och ”mozVisibilityState” är förlegade.  Använd icke-prefixade ”hidden” och ”visibilityState” i stället.
 # LOCALIZATION NOTE: Do not translate "NodeIterator" or "detach()".
 NodeIteratorDetachWarning=Anropet detach() på en NodeIterator har inte längre någon effekt.
-# LOCALIZATION NOTE: Do not translate "Mozilla Audio Data API" and "Web Audio API".
-MozAudioDataWarning=Mozilla Audio Data API är förlegat. Använd Web Audio API i stället.
 # LOCALIZATION NOTE: Do not translate "LenientThis" and "this"
 LenientThisWarning=Ignorerar get eller set av egenskap som har [LenientThis] eftersom objektet ”this” är fel.
 # LOCALIZATION NOTE: Do not translate "nsIDOMWindowUtils", "getWindowWithOuterId", or "nsIWindowMediator"
 GetWindowWithOuterIdWarning=Bruket av nsIDOMWindowUtils.getOuterWindowWithId() är förlegat. Använd i stället metoden nsIWindowMediator för samma namn.
 # LOCALIZATION NOTE: Do not translate "getPreventDefault" or "defaultPrevented".
 GetPreventDefaultWarning=Bruket av getPreventDefault() är förlegat. Använd defaultPrevented i stället.
 # LOCALIZATION NOTE: Do not translate "getUserData", "setUserData", "WeakMap", or "element.dataset".
 GetSetUserDataWarning=Bruket av getUserData() eller setUserData() är förlegat. Använd WeakMap eller element.dataset i stället.
@@ -141,8 +138,14 @@ UseOfReleaseEventsWarning=Bruket av rele
 UseOfDOM3LoadMethodWarning=Bruket av document.load() är förlegat. För att uppgradera din kod, använd i stället DOM-objektet XMLHttpRequest. För mer hjälp, se https://developer.mozilla.org/en/XMLHttpRequest
 # LOCALIZATION NOTE: Do not translate "window.showModalDialog()" or "window.open()" 
 ShowModalDialogWarning=Bruket av window.showModalDialog() är förlegat. Använd window.open() i stället. För mer hjälp, se https://developer.mozilla.org/en-US/docs/Web/API/Window.open
 # LOCALIZATION NOTE: Do not translate "window._content" or "window.content"
 Window_ContentWarning=window._content är förlegat. Använd window.content i stället.
 # LOCALIZATION NOTE: Do not translate "XMLHttpRequest"
 SyncXMLHttpRequestWarning=Synkron XMLHttpRequest på huvudtråden är förlegat på grund av dess negativa inverkan på användarupplevelsen. För mer hjälp, se http://xhr.spec.whatwg.org/
 ImplicitMetaViewportTagFallback=Ingen meta-viewport-tagg hittades. Specificera uttryckligt en för att förhindra oväntade effekter i framtida versioner. För mer hjälp, se https://developer.mozilla.org/en/docs/Mozilla/Mobile/Viewport_meta_tag
+# LOCALIZATION NOTE: Do not translate "DataContainerEvent" or "CustomEvent"
+DataContainerEventWarning=Bruket av DataContainerEvent är förlegat. Använd CustomEvent i stället.
+# LOCALIZATION NOTE: Do not translate "sendAsBinary" or "send(Blob data)"
+SendAsBinaryWarning=Den icke-standardiserade metoden sendAsBinary är förlegad och kommer snart att tas bort. Använd standardmetoden send(Blob data) i stället.
+# LOCALIZATION NOTE: Do not translate "window.controllers"
+Window_ControllersWarning=window.controllers är förlegat. Använd det inte för att identifiera UA.
diff --git a/editor/ui/chrome/composer/editorOverlay.dtd b/editor/ui/chrome/composer/editorOverlay.dtd
--- a/editor/ui/chrome/composer/editorOverlay.dtd
+++ b/editor/ui/chrome/composer/editorOverlay.dtd
@@ -11,17 +11,18 @@
 <!ENTITY pasteNoFormatting.label "Klistra in utan formatering">
 <!ENTITY pasteNoFormatting.accesskey "u">
 <!ENTITY pasteNoFormatting.key "V">
 <!ENTITY pasteAs.label "Klistra in som">
 <!ENTITY pasteAs.accesskey "a">
 <!ENTITY pasteAsQuotationCmd.label "Klistra in som citat">
 <!ENTITY pasteAsQuotationCmd.accesskey "c">
 <!ENTITY pasteAsQuotationCmd.key "o">
-<!ENTITY findCmd.label "Sök och ersätt...">
+<!ENTITY findBarCmd.label "Sök…">
+<!ENTITY findReplaceCmd.label "Sök och ersätt…">
 <!ENTITY enableInlineSpellChecker.label "Stavningskontrollera medan du skriver">
 <!ENTITY enableInlineSpellChecker.accesskey "v">
 <!ENTITY checkSpellingCmd2.label "Stavningskontroll…">
 <!ENTITY checkSpellingCmd2.accesskey "a">
 <!ENTITY checkSpellingCmd2.key "p">
 
 <!-- Insert menu items -->
 <!ENTITY insertMenu.label "Infoga">
@@ -34,16 +35,18 @@
 <!ENTITY insertImageCmd.label "Bild...">
 <!ENTITY insertImageCmd.accesskey "i">
 <!ENTITY insertHLineCmd.label "Vågrät linje">
 <!ENTITY insertHLineCmd.accesskey "r">
 <!ENTITY insertTableCmd.label "Tabell...">
 <!ENTITY insertTableCmd.accesskey "T">
 <!ENTITY insertHTMLCmd.label "HTML...">
 <!ENTITY insertHTMLCmd.accesskey "H">
+<!ENTITY insertMathCmd.label "Formel…">
+<!ENTITY insertMathCmd.accesskey "F">
 <!ENTITY insertCharsCmd.label "Tecken och symboler...">
 <!ENTITY insertCharsCmd.accesskey "c">
 <!ENTITY insertBreakAllCmd.label "Bryt under bild(er)">
 <!ENTITY insertBreakAllCmd.accesskey "y">
 
 <!-- Used just in context popup. -->
 <!ENTITY createLinkCmd.label "Skapa länk...">
 <!ENTITY createLinkCmd.accesskey "p">
diff --git a/security/manager/chrome/pippki/certManager.dtd b/security/manager/chrome/pippki/certManager.dtd
--- a/security/manager/chrome/pippki/certManager.dtd
+++ b/security/manager/chrome/pippki/certManager.dtd
@@ -22,17 +22,17 @@
 <!ENTITY certmgr.detail.general_tab.title     "Allmänt">
 <!ENTITY certmgr.detail.general_tab.accesskey "A">
 <!ENTITY certmgr.detail.prettyprint_tab.title "Detaljer">
 <!ENTITY certmgr.detail.prettyprint_tab.accesskey "D">
 
 <!ENTITY certmgr.pending.label                "Verifierar certifikat…">
 <!ENTITY certmgr.subjectinfo.label            "Utfärdat till">
 <!ENTITY certmgr.issuerinfo.label             "Utfärdat av">
-<!ENTITY certmgr.validity.label               "Giltighet">
+<!ENTITY certmgr.periodofvalidity.label       "Giltighetsperiod" >
 <!ENTITY certmgr.fingerprints.label           "Fingeravtryck">
 <!ENTITY certmgr.certdetail.title             "Certifikatdetaljer">
 <!ENTITY certmgr.certdetail.cn                "Common Name (CN)">
 <!ENTITY certmgr.certdetail.o                 "Organisation (O)">
 <!ENTITY certmgr.certdetail.ou                "Organisationsenhet (OU)">
 <!ENTITY certmgr.certdetail.serialnumber      "Serienummer">
 <!ENTITY certmgr.certdetail.sha1fingerprint   "SHA1-fingeravtryck">
 <!ENTITY certmgr.certdetail.md5fingerprint    "MD5-fingeravtryck">
@@ -54,17 +54,17 @@
 <!ENTITY certmgr.deleteusercert.title         "Ta bort ditt certifikat">
 <!ENTITY certmgr.deleteusercert.beforename    "Du vill ta bort detta certifikat:">
 <!ENTITY certmgr.deleteusercert.aftername     "När du väl har tagit bort detta certifikat kommer du inte att kunna läsa e-post som har krypterats med det.">
 
 <!ENTITY certmgr.certname                     "Certifikatnamn">
 <!ENTITY certmgr.certserver                   "Server">
 <!ENTITY certmgr.override_lifetime            "Livslängd">
 <!ENTITY certmgr.tokenname                    "Säkerhetsenhet">
-<!ENTITY certmgr.issued                       "Utfärdat">
+<!ENTITY certmgr.begins                       "Börjar">
 <!ENTITY certmgr.expires                      "Förfaller">
 <!ENTITY certmgr.email                        "E-postadress">
 <!ENTITY certmgr.serial                       "Serienummer">
 
 <!ENTITY certmgr.close.label                  "Stäng">
 <!ENTITY certmgr.close.accesskey              "ä">
 <!ENTITY certmgr.view2.label                  "Visa…">
 <!ENTITY certmgr.view2.accesskey              "V">
diff --git a/services/sync/sync.properties b/services/sync/sync.properties
--- a/services/sync/sync.properties
+++ b/services/sync/sync.properties
@@ -20,18 +20,16 @@ error.login.description = Sync stötte p
 error.login.prefs.label = Inställningar…
 error.login.prefs.accesskey = I
 # should decide if we're going to show this
 error.logout.title = Fel vid utloggning
 error.logout.description = Sync stötte på ett fel vid anslutningen.  Det är förmodligen OK och du behöver inte gör någonting åt det.
 error.sync.title = Fel vid synkning
 error.sync.description = Sync stötte på ett fel under synkningen: %1$S.  Sync kommer automatiskt att försöka upprepa åtgärden.
 error.sync.prolonged_failure = Synkning har inte kunnat genomföras de senaste %1$S dagarna. Kontrollera dina nätverksinställningar.
-error.sync.no_node_found = Sync-servern är lite upptagen just nu, men du behöver inte göra någonting på grund av detta.  Vi kommer att starta synkningen av din data så fort det är möjligt!
-error.sync.no_node_found.title = Synkfördröjning
 error.sync.serverStatusButton.label = Serverstatus
 error.sync.serverStatusButton.accesskey = v
 error.sync.needUpdate.description = Du behöver uppdatera Firefox Sync för att kunna fortsätta synkningen av din data.
 error.sync.needUpdate.label = Uppdatera Firefox Sync
 error.sync.needUpdate.accesskey = U
 error.sync.tryAgainButton.label = Synka nu
 error.sync.tryAgainButton.accesskey = S
 warning.sync.quota.label = Närmar sig serverkvoten
diff --git a/toolkit/chrome/mozapps/extensions/extensions.dtd b/toolkit/chrome/mozapps/extensions/extensions.dtd
--- a/toolkit/chrome/mozapps/extensions/extensions.dtd
+++ b/toolkit/chrome/mozapps/extensions/extensions.dtd
@@ -97,16 +97,17 @@
 <!ENTITY cmd.alwaysActivate.tooltip           "Använd alltid det här tillägget">
 <!ENTITY cmd.neverActivate.label              "Aktivera aldrig">
 <!ENTITY cmd.neverActivate.tooltip            "Använd aldrig det här tillägget">
 <!ENTITY cmd.stateMenu.tooltip                "Välj när tillägget ska användas">
 <!ENTITY cmd.installAddon.label               "Installera">
 <!ENTITY cmd.installAddon.accesskey           "I">
 <!ENTITY cmd.uninstallAddon.label             "Ta bort">
 <!ENTITY cmd.uninstallAddon.accesskey         "T">
+<!ENTITY cmd.debugAddon.label                 "Avlusa">
 <!ENTITY cmd.showPreferencesWin.label         "Inställningar">
 <!ENTITY cmd.showPreferencesWin.tooltip       "Ändra tilläggets inställningar">
 <!ENTITY cmd.showPreferencesUnix.label        "Inställningar">
 <!ENTITY cmd.showPreferencesUnix.tooltip      "Ändra tilläggets inställningar">
 <!ENTITY cmd.contribute.label                 "Bidra">
 <!ENTITY cmd.contribute.accesskey             "B">
 <!ENTITY cmd.contribute.tooltip               "Ge ett bidrag till utvecklingen av detta tillägg">
 
